src/sparql/ordo-replace-annotation-based-mappings.ru.jinja2

```diff
@@ -0,0 +1,28 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix ECO: <http://purl.obolibrary.org/obo/ECO_>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+prefix sssom: <https://mapping-commons.github.io/sssom/purl/>
+
+{% for mapping_precision_string, relationship_curie in mapping_str__curie__map.items() %}
+DELETE {
+  ?xref_anno a owl:Axiom ;
```

Comment:
Whole axioms were removed.

I was wanting to just remove the `ECO:0000218 ?mapping_precision_string .` property on the axiom.

Now that I think about it, however, these axioms aren't really needed, since they're about class relationships, and I've already inserted those relationships onto the classes.

## Example axiom
```owl
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://www.orpha.net/ORDO/Orphanet_10"/>
        <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasDbXref"/>
        <owl:annotatedTarget>ICD-10:Q98.8</owl:annotatedTarget>
        <obo:ECO_0000218 xml:lang="en">- NTBT (ORPHA code&apos;s Narrower Term maps to a Broader Term).
- Attributed (The ICD10 code is attributed by Orphanet).</obo:ECO_0000218>
    </owl:Axiom>
```

## Questions
1. Was this correct? to remove whole axiom?
2. How do I delete just an axiom property anyway, and not the whole axiom?
I tried just having this as my DELETE clause, but my IDE showed there was a syntax error:
```sparql
DELETE {
    ECO:0000218 ?mapping_precision_string .
```

Reply:
Yes, lets have this removed for now. May cycle back on that later.


to remove just an annotation, you can delete just the triple:

```
?xref_anno ?some ?annotation!
```

Reply:
Ah ok. @matentzn We can mark resolved on this. But first, I just want to confirm my understanding.

So you think that in order just to delete the annotation on the axiom, as opposed to the whole axiom itself, I would change this:
```sparql
DELETE {
  ?xref_anno a owl:Axiom ;
    ECO:0000218 ?mapping_precision_string .
```

which is effectively the same as this:
```sparql
DELETE {
  ?xref_anno a owl:Axiom .
```

to this: 
```sparql
DELETE {
  ?xref_anno ECO:0000218 ?mapping_precision_string .
```

You had an `!` at the end of your example, but I'm guessing that was just there for emphasis and that it would be syntactically correct to have a `.` instead.

Reply:
Thats right!